openid-4-verifiable-presentations-1_0.md

```diff
@@ -611,6 +629,20 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
 
 If the Verifier responds with any HTTP error response, the Wallet MUST terminate the process.
 
+## Attachments in Authorization Requests {#attachments-in-authorization-requests}
+
+The attachment data mechanism enables to pass additional information to the Wallet to request credentials. The Verifier MAY include attachments in the Authorization Request, for example to pass a registration certificate to inform about the registered intended use or an authorization attestation to request a specific credential from the Wallet to comply with the embedded disclosure policy.
+
+### Proof of Possession
+
+In some cases, the Verifier may want to include attachments that are bound to a key or require verification of their origin.
+
+It is up to the ecosystem profile to define how proof of possession is conveyed. Verifiers SHOULD use the `nonce` and `client_id` parameter from the Authorization Request in any proof-of-possession mechanism to bind to this transaction. The Wallet MUST validate such proofs if defined by the profile and ignore or reject attachments that fail validation.
```

Comment:
Please explain you mean the creation and provision of a digital signature as PoP. 

I think the section must zoom out a bit. First of all, it needs to discuss that the wallet must (in one way or the other) be able to bind the attachment to the relying party. There are two ways, either the attachment is bound to attributes confirmed by the client authentication or the client sends another proof of possession for a key included in the attachment. Please add those considerations.